CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 # Change Log
+# [0.64.0 (November 4th, 2020)](https://github.com/eclipse/eclipse.jdt.ls/milestone/82?closed=1)
+ * enhancement - Provide method for converting callstack entry to location. See [#1202](https://github.com/eclipse/eclipse.jdt.ls/issues/1202).
+ * enhancement - Hide inline variable/constant commands when no reference found. See [#1573](https://github.com/eclipse/eclipse.jdt.ls/pull/1573) and [#1575](https://github.com/eclipse/eclipse.jdt.ls/pull/1575).
+ * enhancement - Convert a lambda expression to method reference. See [#1571](https://github.com/eclipse/eclipse.jdt.ls/pull/1571).
+ * bug fix - GTD is not working if referenced library is updated without file name change. See [#1577](https://github.com/eclipse/eclipse.jdt.ls/issues/1577).
+ * bug fix - Fix method ref CompletionItemKind. See [#1574](https://github.com/eclipse/eclipse.jdt.ls/pull/1574).
+ * other - Update Target Platform to Eclipse 2020-12 M1. See [#1567](https://github.com/eclipse/eclipse.jdt.ls/issues/1567).
```

Comment:
missing https://github.com/eclipse/eclipse.jdt.ls/issues/1582

Reply:
Done. I'll add this to vscode-java's Changelog as well.